test/16_recode.jl

```diff
@@ -159,7 +159,21 @@ end
     end
 end
 
-@testset "Recoding array with missings, no default and with missing as a key pair from $(typeof(x)) to $(typeof(y))" for
+@testset "Group recoding array with missings and no default from $(typeof(x)) to $(typeof(y))" for
```

Comment:
Rather than "group recoding", I'd find it more explicit to say "collection in LHS" or "collection with missing in LHS" depending on the case.

Reply:
fixed